tingbot/graphics.py

```diff
@@ -410,7 +395,7 @@ def _get_frames(self, pil_image):
             else:
                 palette = base_palette
 
-            pygame_image = pygame.image.fromstring(pil_image.tostring(), pil_image.size, pil_image.mode)
+            pygame_image = pygame.image.fromstring(pil_image.tobytes(), pil_image.size, pil_image.mode)
```

Comment:
Cool, makes sense. I've got Pillow 1.7.8 on here. Guess we should put version ranges in the setup.py when we do the switch to Python 3. Let's revert this for now, we can address it later.